src/pudl/output/ferc1.py

```diff
@@ -59,9 +59,12 @@ def plants_steam_ferc1(pudl_engine):
         .assign(
             capacity_factor=lambda x: x.net_generation_mwh / (8760 * x.capacity_mw),
             opex_fuel_per_mwh=lambda x: x.opex_fuel / x.net_generation_mwh,
-            opex_nonfuel=lambda x: x.opex_production_total - x.opex_fuel,
+            opex_total_nonfuel=lambda x: x.opex_production_total
+            - x.opex_fuel.fillna(0),
             opex_nonfuel_per_mwh=lambda x: np.where(
-                x.net_generation_mwh > 0, x.opex_nonfuel / x.net_generation_mwh, pd.NA
+                x.net_generation_mwh > 0,
+                x.opex_total_nonfuel / x.net_generation_mwh,
+                pd.NA,
```

Comment:
If this is a `float` column, I think it should be using `np.nan` as the NA value, not `pd.NA` (which is only for the not already nullable types like `string` and `int`)